install/helm/agones/values.yaml

```diff
@@ -36,6 +36,7 @@ agones:
     allocator:
       name: agones-allocator
       annotations: {}
+      labels: {}
```

Comment:
I just realised something! This value hasn't been applied to the actual service Deployment! We will need to do that as well.

https://github.com/googleforgames/agones/blob/main/install/helm/agones/templates/service/allocation.yaml

Reply:
I have added labels in this allocation.yaml file, please share your suggestions